server/src/internalClusterTest/java/org/opensearch/repositories/RepositoriesServiceIT.java

```diff
@@ -149,9 +150,9 @@ public void testCreatSnapAndUpdateReposityCauseInfiniteLoop() throws Interrupted
             repoSettings
         );
 
-        String snapshotName = "test-snapshot";
         Runnable createSnapshot = () -> {
```

Comment:
Maybe better to use a counter? It might make debugging easier.

```java
final AtomicInteger = snapshotCount = new AtomicInteger();
Runnable createSnapshot = () -> {
    String snapshotName = "snapshot-" + snapshotCount.incrementAndGet();
    logger.info("--> beginning snapshot for " + snapshotName);
...
```